### ISSUE TITLE
Is it possible to use multiple OBJ files as animation frames in Sceneform or is there a way to play an FBX animation using shape keys?

### ISSUE THREAD

**Issue by ollyc2015** [google-ar/sceneform-android-sdk#981]
I have a cloth simulation that I am trying to animate whilst in AR and as baking into FBX is not an option as FBX doesn’t work well with mesh transformation (and cloth simulation is all about mesh change), I was looking for an alternative approach.

The only remaining options I could think of are as described in my question, but after searching online, I couldn't seem to find any examples in Android Sceneform that used a similar approach.

Therefore, I would like to know if this is possible and if so, could someone please point me in the right direction or give me an example?

**Comment by ollyc2015** [comment]
I have managed to figure out a way to use multiple OBJ files as animation frames in Sceneform - however, I am still interested to know if there a way to play an FBX animation using shape keys.

**Comment by ollyc2015** [comment]
I decided to switch to Unity as I have been able to achieve my desired result. The closest I got in native Android was creating an animation using Shape Keys in GLTF format, but that still fell short, as Android ARCore doesn't support animations in this format. 

It seems the Model Loader for .obj in the Android Native ARCore isnt optimised that well, since passing through 100 models caused the RAM to ramp up to 1.4GB, then starting an AR session killed the app! The same approch in Unity sat at around 350MB RAM after loading and running 100 .obj  animation frames!
